proxy/router/src/lib.rs

```diff
@@ -228,6 +274,7 @@ where
             Error::Route(ref why) =>
                 write!(f, "route recognition failed: {}", why),
             Error::NotRecognized => f.pad("route not recognized"),
+            Error::OutOfCapacity => f.pad("router out of capacity"),
```

Comment:
TIOLI: might be worthwhile for the `OutOfCapacity` error to carry the capacity that was exceeded, since we expect to have routers with differing capacities? That might make the error message more useful.